Milyli.ScriptRunner.Services.Interfaces/Models/Requests/CreateScriptRunRequest.cs

```diff
@@ -0,0 +1,38 @@
+﻿namespace Milyli.ScriptRunner.Services.Interfaces.Models.Requests
+{
+	using System;
+	using System.Collections.Generic;
+
+	public class CreateScriptRunRequest
+	{
+		public int Id { get; set; }
+
+		public int RelativityScriptId { get; set; }
+
+		public int WorkspaceId { get; set; }
+
+		public string WorkspaceName { get; set; }
+
+		public string Name { get; set; }
+
+		public DateTime? LastExecutionTime { get; set; }
+
+		public DateTime? NextExecutionTime { get; set; }
+
+		public int JobStatus { get; set; }
+
+		public bool JobEnabled { get; set; } = true;
+
+		/// <summary>
+		/// Gets or sets the bitmask that represents the schedule.  Only the first 7 bits (0x01 through 0x7F) are used, the LSB represents Sunday, the 7th bit represents Saturday
+		/// </summary>
+		public int ExecutionSchedule { get; set; }
```

Comment:
Is there a way we can not do this for the Kepler Services? If we store it internally as an int still that's fine but we should expose something easier to use.

- [ ] Try to avoid having the end user bit-shift